google/resource_bigquery_dataset_test.go

```diff
@@ -112,3 +159,97 @@ resource "google_bigquery_dataset" "test" {
   }
 }`, datasetID)
 }
+
+func testAccBigQueryDatasetWithOneAccess(datasetID string) string {
+	return fmt.Sprintf(`
+resource "google_bigquery_dataset" "access_test" {
+  dataset_id                  = "%s"
```

Comment:
nit: since this isn't trying to align with anything else, you can move the equals sign way back over to the left